.changeset/four-hounds-own.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-techdocs': minor
+'@backstage/plugin-techdocs-backend': minor
+---
+
+When newer documentation available but not built, show older documentation while async building newer
```

Comment:
We should also mention the new and removed endpoints.

Reply:
Updated